dashboards/scripts/export_dashboard_json.py

```
@@ -0,0 +1,134 @@
+    if not output_file:
+        output_file = os.path.join(output_dir, f"dashboard_{args.dashboard_id}.json")
+    else:
+        # If user provides a path, ensure we can write to it
+        os.makedirs(os.path.dirname(os.path.abspath(output_file)) or '.', exist_ok=True)
+
+    with open(output_file, "w") as f:
+        # Format the JSON nicely before saving
+        try:
+            parsed_json = json.loads(json_content)
+            json.dump(parsed_json, f, indent=2)
+        except:
```

Ruff (0.15.5)

[error] Do not use bare `except`

(E722)